packages/components/src/navigation/styles/navigation-styles.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+const horizontalPadding = '30px';
+
+export const Root = styled.div`
+	width: 272px;
+	padding: 35px 0px;
+	> *:not( .components-navigation__menu ) {
+		margin-left: ${ horizontalPadding };
+		margin-right: ${ horizontalPadding };
+	}
+`;
+
+export const Menu = styled.ul`
+	padding: 0;
+	margin: 0 0 32px 0;
+	display: flex;
+	flex-direction: column;
+`;
+
+export const MenuItem = styled.li`
+	display: flex;
```

Comment:
Does the `Flex` component offer something here?

Reply:
It was originally meant to help vertically center items.  I don't think we need it since we're using buttons which are already `inline-flex`.